src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp/Functions/ReindexSearchDocuments/ReindexSearchDocumentsFunction.cs

```diff
@@ -0,0 +1,24 @@
+﻿using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Clients.AzureSearch;
+using GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.CreateSearchableReleaseDocuments.Dtos;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Functions.ReindexSearchDocuments;
+
+public class ReindexSearchDocumentsFunction(
```

Comment:
I think `ReindexSearchableDocumentsFunction` would be more in keeping with the naming elsewhere because we've used 'searchable documents' in quite a few places.